util/measures/2018/quality-measures.csv

```diff
@@ -1 +1 @@
Cataracts: Complications within 30 Days Following Cataract Surgery Requiring Additional Surgical Procedures,CMS132v6,N/A,564,192,"Percentage of patients aged 18 years and older with a diagnosis of uncomplicated cataract who had cataract surgery and had any of a specified list of surgical procedures in the 30 days following cataract surgery which would indicate the occurrence of any of the following major complications: retained nuclear fragments, endophthalmitis, dislocated or wrong power IOL, retinal detachment, or wound dehiscence",Patient Safety,Outcome,Physician Consortium for Performance Improvement,-,-,X,-,-,X,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,singlePerformanceRate,2017,NULL,,TRUE,TRUE,NULL,1,1,,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_192_Registry.pdf,,,https://ecqi.healthit.gov/ecqm/measures/cms132v6,FALSE,FALSE
Radiology: Stenosis Measurement in Carotid Imaging Reports ,N/A,N/A,507,195,"Percentage of final reports for carotid imaging studies (neck magnetic resonance angiography [MRA], neck computed tomography angiography [CTA], neck duplex ultrasound, carotid angiogram) performed that include direct or indirect reference to measurements of distal internal carotid diameter as the denominator for stenosis measurement",Effective Clinical Care,Process,American College of Radiology,X,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,singlePerformanceRate,2017,NULL,,FALSE,FALSE,1,1,NULL,,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_195_Registry.pdf,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_195_Claims.pdf,,,FALSE,FALSE
Ischemic Vascular Disease (IVD): Use of Aspirin or Another Antithrombotic,CMS164v6,N/A,68,204,"Percentage of patients 18 years of age and older who were diagnosed with acute myocardial infarction (AMI), coronary artery bypass graft (CABG) or percutaneous coronary interventions (PCI) in the 12 months prior to the measurement period, or who had an active diagnosis of ischemic vascular disease (IVD) during the measurement period, and who had documentation of use of aspirin or another antiplatelet during the measurement period",Effective Clinical Care,Process,National Committee for Quality Assurance,X,-,X,X,-,X,-,-,X,-,-,-,-,X,X,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,-,singlePerformanceRate,2017,NULL,,FALSE,FALSE,1,1,1,,,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_204_Registry.pdf,https://qpp.cms.gov/docs/QPP_quality_measure_specifications/Claims-Registry-Measures/2018_Measure_204_Claims.pdf,https://qpp.cms.gov/docs/QPP_quality_measure_spec
```

Comment:
wow, the csv being one line really makes it impossible to review changes. this wasn't always the case right? hm...